results.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="example" tests="2" failures="0" time="2.0">
+  <testsuite name="Hello Automation">
+    <testcase classname="Hello Automation" name="Test Case One" time="1.00"/>
+    <testcase classname="Hello Automation" name="Test Case Two" time="1.00"/>
```

Comment:
Does it make sense to have one of them failed?
```
<testcase classname="Hello Automation" name="Test Case Two" time="1.00">
  <failure type="Exception" message="Something unexpected happened"/>
</testscase>
```

Also, should you consider moving that `results.xml` from the root and putting it is folder, e.g. `reports`?

Reply:
Good suggestion. I will add this on the next iteration, once exploratory is available. All of the screen images will require updating.